packages/sync/src/Settings.php

```diff
@@ -47,12 +47,15 @@ class Settings {
 		'comment_meta_whitelist'                 => true,
 		'max_enqueue_full_sync'                  => true,
 		'max_queue_size_full_sync'               => true,
+		'full_sync_send_immediately'             => true,
```

Comment:
This is a duplicate line.

Reply:
https://github.com/Automattic/jetpack/pull/13963/commits/639e648e8bd42ac481fb1a99b7cc486e30b5c64e